contracts/lib/CircularBufferUint.sol

```diff
@@ -47,6 +47,11 @@ contract CircularBufferUint {
      *
      * @param _maxItems Defines how many items this buffer stores before
      *                  overwriting older items.
+     *
+     * \pre The function can be called only once.
+     * \pre `_maxItems` must be greater than 0.
+     *
+     * \post Sets length of `items` with the value of `_maxItems`.
```

Comment:
Here we are exposing private variable information. The previous comment was better as to me. What do you think?

Reply:
True, it is exposing too much information about private variable
Hence, i have reverted the change.